cypress/integration/pages/topicPage/tests.js

```diff
@@ -15,42 +15,36 @@ export default ({ service, pageType, variant }) => {
     beforeEach(() => {
       cy.log(Cypress.env('currentPath'));
       cy.log(service);
-      const env = Cypress.env('APP_ENV');
-      if (env !== 'local') {
-        // eslint-disable-next-line prefer-destructuring
-        topicId = Cypress.env('currentPath')
-          .split('topics/')
-          .pop()
-          .split('?')[0];
+      // eslint-disable-next-line prefer-destructuring
+      topicId = Cypress.env('currentPath').split('topics/').pop().split('?')[0];
```

Comment:
You can fix this linting error with the following:
```suggestion
      [topicId] = Cypress.env('currentPath').split('topics/').pop().split('?');
```